pkg/workload/admissionchecks.go

```diff
@@ -139,8 +137,8 @@ func RejectedChecks(wl *kueue.Workload) []kueue.AdmissionCheckState {
 
 // HasAllChecksReady returns true if all the checks of the workload are ready.
 func HasAllChecksReady(wl *kueue.Workload) bool {
-	for i := range wl.Status.AdmissionChecks {
-		if wl.Status.AdmissionChecks[i].State != kueue.CheckStateReady {
+	for _, ac := range wl.Status.AdmissionChecks {
```

Comment:
nit: I prefer to revert to avoid distractions

Reply:
Done